src/System.Memory/tests/Memory/MemoryManager.cs

```diff
@@ -48,10 +48,6 @@ public static void MemoryManagerCtorInvalid()
             Assert.Throws<ArgumentOutOfRangeException>(() => new Memory<int>(manager, 0, -1));
             Assert.Throws<ArgumentOutOfRangeException>(() => new Memory<int>(manager, -1, 0));
             Assert.Throws<ArgumentOutOfRangeException>(() => new Memory<int>(manager, -1, -1));
-            Assert.Throws<ArgumentOutOfRangeException>(() => new Memory<int>(manager, -1, -1));
```

Comment:
Why only remove these specific test cases? What about other tests that use the removed (now internal) `Memory<T>` ctor?

Reply:
Yeah, will rebase this back out; have the changes wrong

Reply:
Fixed